WordPressAuthenticator/Sources/Signin/LoginLinkRequestViewController.swift

```diff
@@ -38,18 +38,31 @@ class LoginLinkRequestViewController: LoginViewController {
         let email = loginFields.username
         if email.isValidEmail() {
             Task {
-                try await gravatarView?.setGravatarImage(with: email, rating: .x)
+                try await downloadAvatar()
             }
         } else {
             gravatarView?.isHidden = true
         }
+        NotificationCenter.default.addObserver(self, selector: #selector(refreshAvatar), name: .GravatarQEAvatarUpdateNotification, object: nil)
```

Comment:
`viewWillAppear` may get called multiple times. I don't think this line is meant to be called multiple times?